src/rules/padding.ts

```diff
@@ -14,8 +14,8 @@ import { AST, Rule, SourceCode } from 'eslint';
 import { Node } from 'estree';
 import * as astUtils from '../ast-utils';
 
-// Statement types we'll respond to
-export const enum StatementType {
+// eslint-disable-next-line no-shadow
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/issues/2484#issuecomment-687257773
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-shadow.md

@dangreenisrael @MichaelDeBoey The fix for this is to update the project's linting config as described in links above. It's an erroneous no-shadow error in this case. We should have been using the typescript no-shadow rule. Sorry about that @MichaelDeBoey... probably seemed pretty weird to see it shadowing itself on the same line.

Reply:
(My mistake originally. Shame!)